packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/sync/SyncSession.kt

```diff
@@ -73,6 +74,21 @@ public interface SyncSession {
      */
     public val state: State
 
+    /**
+     * The [SyncConfiguration] responsible for controlling the session.
+     */
+    public val configuration: SyncConfiguration
+
+    /**
+     * The [User] defined by the [SyncConfiguration] to authenticate the session on Atlas App Services.
```

Comment:
```suggestion
     * The [User] used to authenticate the session on Atlas App Services.
```

I would reword this slightly to just include the most important bits.